### PR TITLE
Feat/implement head

### PR DESCRIPTION
**Pull request by filippov0705**

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -1,18 +1,13 @@
...
 import { connect } from 'react-redux';
-import { UserMenu } from './UserMenu';
-import { HeaderBar } from './HeaderBar';
 import { menuStateChange } from '../action/HeaderActions';
+import SearchAppBar from './SearchAppBar'
 
 class Header extends Component {
   render() {
       return (
         <React.Fragment>
```
Comment (SerhGitHub) [inline src/components/Header.js]: You don't need use React.Fragment here.

File: src/components/SearchAppBar.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+          <IconButton
+            edge="start"
+            className={classes.menuButton}
+            color="inherit"
+            aria-label="open drawer"
+          >
+            <MenuIcon />
+          </IconButton>
+          <Typography className={classes.title} variant="h6" noWrap>
+            Material-UI
```
Comment (SerhGitHub) [inline src/components/SearchAppBar.js]: You should change title.

File: src/containers/App.js [hunk 3]
```diff
@@ -1,22 +1,12 @@
...
-
-
-    return (
-      <React.Fragment>
-
-          <Main />
-
-          </React.Fragment>
-    )
+    return <Main />
```
Comment (SerhGitHub) [inline src/containers/App.js]: Will you add to App more components? I asked you because you create App component and use only Main inside. Maybe you should use Main component instead of the App component.

File: src/components/SearchAppBar.js [hunk 4]
```diff
@@ -79,7 +87,19 @@ class SearchAppBar extends Component {
             <MenuIcon />
           </IconButton>
           <Typography className={classes.title} variant="h6" noWrap>
-            Material-UI
+            Some Logo
```
Comment (SerhGitHub) [inline src/components/SearchAppBar.js]: What do you mean? What is 'Some Logo'? Is it image?

File: src/components/MenuBar.js [hunk 5]
```diff
@@ -2,36 +2,52 @@ import React, { Component } from 'react';
...
-                 <MenuItem onClick={popupState.close}>Sign out</MenuItem>
-                </Menu>
-             </React.Fragment>
-         )}
-        </PopupState>
-        );
-    }
+
+
+class SimpleMenu extends Component {
```
Comment (SerhGitHub) [inline src/components/MenuBar.js]: I don't like "SimpleMenu" name. Please try to find a better component name.

File: src/components/Footer.js [hunk 6]
```diff
@@ -1,9 +1,20 @@
...
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
 
 export default class Footer extends React.Component {
     render() {
         return (
-        <footer className='footer'></footer>
+            <React.Fragment>
+            <CssBaseline />
+            <Container maxWidth="xl" style={{ backgroundColor: '#24292e', height: '130px' }}>
```
Comment (SerhGitHub) [inline src/components/Footer.js]: Don't use inline styles!

File: src/components/Footer.js [hunk 7]
```diff
@@ -9,10 +9,28 @@ export default class Footer extends React.Component {
         return (
             <React.Fragment>
             <CssBaseline />
-            <Container maxWidth="xl" style={{ backgroundColor: '#24292e', height: '130px' }}>
-              <Typography component="div">
-                <Grid></Grid>
-              </Typography>
+            <Container 
+              maxWidth="xl" 
+              style={{
```
Comment (SerhGitHub) [inline src/components/Footer.js]: Don't use inline styles!

File: src/components/Footer.js [hunk 8]
```diff
@@ -9,10 +9,28 @@ export default class Footer extends React.Component {
...
-                <Grid></Grid>
-              </Typography>
+            <Container 
+              maxWidth="xl" 
+              style={{ 
+                backgroundColor: '#24292e', 
+                height: '130px' }}>
+                <Typography 
+                  component="div" 
+                  style={{width: '100%',
```
Comment (SerhGitHub) [inline src/components/Footer.js]: Don't use inline styles!

File: src/components/Footer.js [hunk 9]
```diff
@@ -9,10 +9,28 @@ export default class Footer extends React.Component {
...
+                  alignItems: 'center'
+                  }} >
+                    <Grid container>
+                      <Grid item xs={3}  style={{backgroundColor: 'green'}}>
+                        
+                      </Grid>
+                      <Grid item xs={3}  style={{backgroundColor: 'green'}}>asdasd</Grid>
+                      <Grid item xs={3}  style={{backgroundColor: 'green'}}>asdasd</Grid>
+                      <Grid item xs={3}  style={{backgroundColor: 'green'}}>asdasd</Grid>
+                    </Grid>
```
Comment (SerhGitHub) [inline src/components/Footer.js]: Don't use inline styles!

File: src/components/Header.js [hunk 10]
```diff
@@ -1,11 +1,12 @@
...
 import { connect } from 'react-redux';
 import { menuStateChange } from '../action/HeaderActions';
-import SearchAppBar from './SearchAppBar'
+// import SearchAppBar from './SearchAppBar'
+import PrimarySearchAppBar from './PrimarySearchAppBar'
 
 class Header extends Component {
   render() {
-      return <SearchAppBar />
+      return <PrimarySearchAppBar />
```
Comment (SerhGitHub) [inline src/components/Header.js]: What's the difference between SearchAppBar and PrimarySearchAppBar?

File: src/components/PrimarySearchAppBar.js [hunk 11]
```diff
@@ -0,0 +1,239 @@
...
+      id={menuId}
+      keepMounted
+      transformOrigin={{ vertical: 'top', horizontal: 'right' }}
+      open={isMenuOpen}
+      onClose={this.handleMenuClose}
+    >
+      <MenuItem onClick={this.handleMenuClose}>Profile</MenuItem>
+      <MenuItem onClick={this.handleMenuClose}>My account</MenuItem>
+    </Menu>
+  );
```
Comment (SerhGitHub) [inline src/components/PrimarySearchAppBar.js]: Please check indents in this block.

File: src/components/PrimarySearchAppBar.js [hunk 12]
```diff
@@ -0,0 +1,239 @@
...
+                root: classes.inputRoot,
+                input: classes.inputInput,
+              }}
+              inputProps={{ 'aria-label': 'search' }}
+            />
+          </div>
+          <div className={classes.grow} />
+          <div className={classes.sectionDesktop}>
+            <Link to={mainPath()} >
+            <Button variant="contained" style={{marginRight: '5px'}}>
```
Comment (SerhGitHub) [inline src/components/PrimarySearchAppBar.js]: Don't use inline styles!

File: src/components/Footer.js [hunk 13]
```diff
@@ -24,7 +24,7 @@ export default class Footer extends React.Component {
                   }} >
                     <Grid container>
                       <Grid item xs={3}  style={{backgroundColor: 'green'}}>
-                        
+                     
                       </Grid>
                       <Grid item xs={3}  style={{backgroundColor: 'green'}}>asdasd</Grid>
                       <Grid item xs={3}  style={{backgroundColor: 'green'}}>asdasd</Grid>
```
Comment (SerhGitHub) [inline src/components/Footer.js]: Don't commit temporary code with comments!

File: src/components/Footer.js [hunk 14]
```diff
@@ -3,10 +3,22 @@ import CssBaseline from '@material-ui/core/CssBaseline';
...
+    color: 'white'
+  }
+})
+
+const Footer = props => {
+  const { classes } = props;
+
+  return (
             <React.Fragment>
             <CssBaseline />
```
Comment (SerhGitHub) [inline src/components/Footer.js]: You should add indent.

File: src/components/Footer.js [hunk 15]
```diff
@@ -23,16 +35,28 @@ export default class Footer extends React.Component {
...
+                        <span>Телефоны:</span>
+                        <span>+375 33 38 76 163</span>
+                        <span>+375 29 15 71 703</span>
+                      </Grid>
+                      <Grid item xs={4} className={classes.columnGrid}>
+                        <span>E-mail:</span>
+                        <a href="mailto:careers@itechart-group.com">careers@itechart-group.com</a>
+                        <span>Старт карьеры:</span>
+                        <a href="mailto:st.lab@itechart-group.com">st.lab@itechart-group.com</a>
+                        <span>Сотрудничество:</span>
```
Comment (SerhGitHub) [inline src/components/Footer.js]: You should implement supporting multiple languages.

File: src/containers/Procedures.js [hunk 16]
```diff
@@ -47,7 +47,6 @@ class Procedures extends Component {
               <Typography 
                 component="div" 
                 style={{ 
-                  marginTop: '64px',
                   backgroundColor: '#cfe8fc', 
                   height: '78vh', 
                   width: '100%',
```
Comment (SerhGitHub) [inline src/containers/Procedures.js]: Don't use inline styles!

File: src/containers/Procedures.js [hunk 17]
```diff
@@ -74,6 +73,9 @@ class Procedures extends Component {
                         </Button>
                     </Grid>
                 </Grid>
+                <Grid container style={{}}>
+
+                </Grid>
```
Comment (SerhGitHub) [inline src/containers/Procedures.js]: Why did you add this block?

File: src/components/MenuBar.js [hunk 18]
```diff
@@ -17,7 +17,7 @@ const styles = theme => ({
   },
 });
 
-class SimpleMenu extends Component {
+class HeaderUserMenu extends Component {
```
Comment (SerhGitHub) [inline src/components/MenuBar.js]: Rename 'HeaderUserMenu' to 'Menu'. Your file name and class name should be consistency.

File: src/components/ProceduresAndTasksBar.js [hunk 19]
```diff
@@ -0,0 +1,17 @@
...
+
+const ProceduresAndTasksBar = props => (
+<Grid 
+  item 
+  style={{
+    height: '665px', 
+    backgroundColor: 'white', 
+    margin: '20px',
+    border: '1px solid gray',
+    borderRadius: '7px'}}>
```
Comment (SerhGitHub) [inline src/components/ProceduresAndTasksBar.js]: Don't use inline styles!

File: src/components/ProceduresAndTasksBar.js [hunk 20]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import Grid from '@material-ui/core/Grid';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+const ProceduresAndTasksBar = props => (
```
Comment (SerhGitHub) [inline src/components/ProceduresAndTasksBar.js]: What is ProceduresAndTasksBar? I don't understand.

File: src/components/ProcedureTask.js [hunk 21]
```diff
@@ -0,0 +1,46 @@
...
+const ProcedureTask = props => {
+    const { classes } = props;
+
+    return (
+        <Grid container 
+        style={{
+            width: '100%',
+            height: '40px',
+            borderBottom: '1px solid rgba(94, 92, 92, 0.225)',
+        }}>
```
Comment (SerhGitHub) [inline src/components/ProcedureTask.js]: Don't use inline styles!

File: src/components/ProcedureTask.js [hunk 22]
```diff
@@ -0,0 +1,46 @@
...
+            <Grid item xs={9} className={classes.grid}>
+                <span className={classes.gridSpan}>Procedure 1</span>
+            </Grid>
+            <Grid 
+                item 
+                xs={3} 
+                className={classes.grid} 
+                style={{justifyContent: 'space-around'}}>
```
Comment (SerhGitHub) [inline src/components/ProcedureTask.js]: Don't use inline styles!

File: src/components/Footer.js [hunk 23]
```diff
@@ -25,15 +25,15 @@ const Footer = props => {
...
-                  height: '130px',
-                  display: 'flex',
-                  alignItems: 'center'
-                  }} >
+                  style={{
+                    width: '100%', 
+                    maxWidth: '1200px', 
+                    display: 'flex',
+                    flexDirection: 'column',
+                    alignItems: 'center'}} >
```
Comment (SerhGitHub) [inline src/components/Footer.js]: Don't use inline styles!

File: src/components/Footer.js [hunk 24]
```diff
@@ -25,15 +25,15 @@ const Footer = props => {
...
                     <Grid container>
                       <Grid item xs={4}   className={classes.columnGrid}>
                         <img src={FooterLogo} alt="logo" style={{width: '150px'}} />
```
Comment (SerhGitHub) [inline src/components/Footer.js]: Don't use inline styles!

File: src/components/PopUpWindow.js [hunk 25]
```diff
@@ -0,0 +1,96 @@
...
+  paper: {
+    backgroundColor: theme.palette.background.paper,
+    border: '2px solid #000',
+    boxShadow: theme.shadows[5],
+    padding: theme.spacing(2, 4, 3),
+  },
+});
+
+
+
```
Comment (SerhGitHub) [inline src/components/PopUpWindow.js]: Remove two lines, please

File: src/components/PopUpWindow.js [hunk 26]
```diff
@@ -0,0 +1,96 @@
...
+            case 'Edit':
+                return <EditRoundedIcon onClick={this.handleOpen} />
+
+            case 'More':
+                return <MoreHorizRoundedIcon onClick={this.handleOpen} />
+
+            case 'Play': 
+                return <PlayCircleFilledWhiteRoundedIcon onClick={this.handleOpen} />
+            default:
```
Comment (SerhGitHub) [inline src/components/PopUpWindow.js]: Formerly, you leave a line before new 'case' but before 'default' you didn't leave a line. You should choose one way.

Comment (SerhGitHub) [inline src/components/PopUpWindow.js]: Formerly, you leave a line before new 'case' but before 'default' you didn't leave a line. You should choose one way.

File: src/components/PopUpWindow.js [hunk 27]
```diff
@@ -0,0 +1,96 @@
...
+                return null
+        }
+    }
+
+    render() {
+const { classes, data } = this.props,
+    { open } = this.state;
+
+  return (
+    <div style={{marginTop: '5px'}}>
```
Comment (SerhGitHub) [inline src/components/PopUpWindow.js]: Don't use inline styles!

File: src/components/ProcedureTask.js [hunk 28]
```diff
@@ -17,6 +17,37 @@ const styles = theme => ({
 const ProcedureTask = props => {
```
Comment (SerhGitHub) [inline src/components/ProcedureTask.js]: What's 'ProcedureTask'? Is it 'Procedure' or 'Task'?

File: src/components/ProcedureTask.js [hunk 29]
```diff
@@ -17,6 +17,37 @@ const styles = theme => ({
...
     const { classes } = props;
 
+    function getProcedureContent() {
+        return (
+            <Grid 
+             item 
+             xs={3} 
+             className={classes.grid} 
+             style={{justifyContent: 'space-around'}}>
```
Comment (SerhGitHub) [inline src/components/ProcedureTask.js]: Don't use inline styles!

File: src/components/ProcedureTask.js [hunk 30]
```diff
@@ -17,6 +17,37 @@ const styles = theme => ({
...
+        )
+    }
+
+    function getTaskContent() {
+        return (
+            <Grid 
+             item 
+             xs={3} 
+             className={classes.grid} 
+             style={{justifyContent: 'space-around'}}>
```
Comment (SerhGitHub) [inline src/components/ProcedureTask.js]: Don't use inline styles!

Comment (SerhGitHub) [inline src/components/ProcedureTask.js]: Maybe it made more sense to separate 'ProcedureTask', 'getProcedureContent' and 'getTaskContent' on three components.

File: src/components/ProcedureTask.js [hunk 31]
```diff
@@ -25,9 +56,11 @@ const ProcedureTask = props => {
             borderBottom: '1px solid rgba(94, 92, 92, 0.225)',
         }}>
             <Grid item xs={9} className={classes.grid}>
-                <span className={classes.gridSpan}>Procedure 1</span>
+                <span className={classes.gridSpan}>{props.data} 1</span>
```
Comment (SerhGitHub) [inline src/components/ProcedureTask.js]: Remove fake data from common component.

File: src/components/ProcedureTask.js [hunk 32]
```diff
@@ -17,6 +17,37 @@ const styles = theme => ({
...
+                <TransitionsModal data={'More'} />
+                <PlayCircleFilledWhiteRoundedIcon /> */}
+                
+            </Grid>
+        )
+    }
+
     return (
         <Grid container 
         style={{
```
Comment (SerhGitHub) [inline src/components/ProcedureTask.js]: Don't use inline styles.

File: src/components/MenuBar.js (hunk 18 above)
Comment (filippov0705) [reply src/components/MenuBar.js]: changed to UserMenu
